build/openwisp_base/Dockerfile

```diff
@@ -43,7 +43,8 @@ ARG OPENWISP_UTILS_SOURCE=https://github.com/openwisp/openwisp-utils/tarball/doc
 RUN pip install --prefix='/install' --upgrade --force-reinstall ${OPENWISP_UTILS_SOURCE}
 RUN pip install --prefix='/install' channels_redis service_identity django-redis psycopg2 \
                                     uwsgi sentry-sdk supervisor celery django-celery-beat \
-                                    django-cors-headers django-rest-auth jinja2
+                                    django-cors-headers django-rest-auth jinja2 \
+                                    django-pipeline~=2.0.0 django-compress-staticfiles~=1.0.1b
```

Comment:
`django-compress-staticfiles~=1.0.1b` dependency is already defined in [openwisp-utils]( https://github.com/openwisp/openwisp-utils/blob/c5558fc14a22316e49e9144d39216de348e19b05/setup.py#L46).

We can skip here.